matchms/Scores.py

```diff
@@ -128,39 +128,46 @@ def calculate(self) -> Scores:
                                                            is_symmetric=self.is_symmetric)
         return self
 
-    def scores_by_reference(self, reference: ReferencesType) -> numpy.ndarray:
+    def scores_by_reference(self, reference: ReferencesType,
+                            sort: bool = False) -> numpy.ndarray:
         """Return all scores (not sorted) for the given reference spectrum.
```

Comment:
Sorting is explained in sort parameter so no need to have it in the description.
```suggestion
        """Return all scores for the given reference spectrum.
```